### ISSUE TITLE
Problem with sameas and concepticon test

### ISSUE THREAD

**Issue by Schweikhard** [clld/concepticon-data#500]
The concept of legendary creature is redundant as we also have mythical being.
When I add into the conceptrelations.tsv the line

2138 | LEGENDARY CREATURE | sameas | 2641 | MYTHICAL BEING
-- | -- | -- | -- | --

concepticon test tells me that there is a deprecated concept in Chacon 2014. However, that conceptlist is only linked to mythical being and not to legendary creature in my fork, so I don't know what causes the error.
Also swapping the concepts in conceptrelations.tsv doesn't solve the problem, nor does adding the replacement ID in the concepticon.tsv to either. What am I missing?

**Comment by LinguList** [comment]
SAMEAS relations will default to the lower ID, so you need to link Chacon ot the smaller ID now.

**Comment by Schweikhard** [comment]
Oh, good to know. Would it also be possible to simply remove the redundant concept from the list? It's not used by any concept list as far as I see.

**Comment by LinguList** [comment]
no, look please at our concepticon.tsv, you see that we have an extra column that deprecates a concept, so you have to add the ID of the smaller ID there. No deletion possible.

**Comment by Schweikhard** [comment]
So it's because we can't skip a number? Then I would just replace that unused number with a new concept. Or I'll just swap the names of the two concepts, then I can at least use the concept name of "mythical being" as that is also the one used by Chacon.

**Comment by LinguList** [comment]
NOPE. This will prevent backwards compatibility with older versions. We have decided this explicitly, please check the last column in concepticon.tsv to see similar cases for htis.

**Comment by Schweikhard** [comment]
I understand, didn't think of that. So also renamings should be only done via "same as" and not by editing the name in concepticon.tsv?

**Comment by LinguList** [comment]
The idea stays the same, so odn't worry too much now about that. But try to add the higher ID to the "REAPLACMENENT_ID" field, for chacon and see if it throws an error.
